crates/agglayer-node/src/rpc/mod.rs

```diff
@@ -16,7 +16,8 @@ use jsonrpsee::{
         ErrorObject, ErrorObjectOwned,
     },
 };
-use pessimistic_proof::certificate::Certificate;
+use pessimistic_proof::batch_header::BatchHeader;
```

Comment:
I think we need to rethink the name `BatchHeader` it's already caused some confusion with the CDK team

Reply:
I don't have a strong opinion on this. What name should we change it to?

Reply:
cc @krlosMata @zuiris
Is `MultiBatchHeader` ok with you to replace `Certificate`? Or do you have any other name idea? It refers to the data structure submitted to the AggLayer

Reply:
Why don't we go with `MultiBatchHeader` for now to get this merged? We have a while where we could change it if someone has a "better" idea anyway.

Reply:
Renamed to `MultiBatchHeader` in 2730558.

Reply:
I don't find this discussion particularly productive tbh, I'm pretty sure there is a better application for everyone's brain capacity than having a discussion about how to name a data structure. If we want to optimize for "less confusing" name, `certificate` is less confusing IMO because it is at least signals of a new entity in a protocol, which can't be said about any name, which includes "batch" in it.

Reply:
I totally agree on what @zuiris are saying. We already had discussions about that during meetings when discussing the specification and we end up agreeing on `Certificate`, as it was less confusing with other structs and concept used by other components in relation with the `agglayer`. I don't have a strong opinion on what we should use, but I would like to have something on which we can rely on without have to add extra context.
I may not have the full picture, but does `MultiBatchHeader` mean that we can somehow get a `BatchHeader` or even a `Batch` using this struct ? Does it mean that we can receive multiple instances of what we were calling `Certificate` in one RPC call ?

Reply:
@zuiris agree that there are more productive discussions than naming. But actually thinking about the future direction of the protocol is probably a very good use of brainpower. 

If we project a tiny bit into the future, what will we call the data structure that we use to authenticate a signing key for a chain, or a relay, or some other service? This is what the term "Certificate" actually refers to. 

>  it is at least signals of a new entity in a protocol

This is not correct and (imo) signals a misunderstanding of the protocol. I suggested BatchHeader[1] initially because it would be better, imo, for the verifier to use the same header as a public input for the pessimistic proof and (aggregated) execution and consensus proofs, because it's simpler and we're less likely to mess up by not checking that two inputs are consistent. The point is that we should be committing to the same public inputs across proofs to avoid errors.

Like I said in my above comment, if we're carrying technical debt that doesn't allow us to call any sequence of blocks a "Batch" then we can call it something else. I want to be clear though, the point here is not to be pedantic, it's to ensure that we share a consistent understanding of the protocol and how it's supposed to work.

[1] You might notice that the data structure in the PP has other fields, like ImportedBridgeExits, that are included in plaintext but don't belong in a "Header". I had left comments that these should eventually be moved to separate structs for auxiliary data that we provide as private inputs.